02/List.js

```diff
@@ -1,13 +1,9 @@
-import React from 'react';
-
-import ListItem from './ListItem';
+import React from "react";
 
 function List(props) {
-    return (
-        <ul>
-            <ListItem />
-        </ul>
-    )
+	const { children } = props;
+
+	return <ul>{children}</ul>;
```

Comment:
Osobiście zrobiłbym tu pętlę, która wyświetla elementy ponieważ to komponent, który zwiera wiele "item" i to w nim zawarłbym logikę dotyczącą prezentowania wielu elemetnów ;)